### DISCUSSION TITLE
Support settings overriding custom support shape.

### DISCUSSION THREAD

**Discussion by Deadit3** [5axes/CustomSupportCylinder#28] [Q&A] [answered]
I turned off cura supports but the pattern setting there overrides the cylinders I want. Updating the plugin didn't fix issue. I was still using 4.13 but updating to 5.2.2 didn't fix either.
![image](https://user-images.githubusercontent.com/113082660/216265694-1fbf63c5-c907-4958-a2bc-317053f9925f.png)
![image](https://user-images.githubusercontent.com/113082660/216265727-73ceff94-995a-4e7a-bf3c-b8c83114981b.png)

**Comment by 5axes** [comment]
This is obviously not a plugin problem. The purpose of the plugin is only to create the supports in CAD form. The creation of the toolpath is a Cura issue. In this case we don't have enough information about your parameters to be able to diagnose your problem. The easiest way is to share your project by zipping the project file.

**Comment by Deadit3** [comment]
Ok here's the .3mf file, btw thanks for the quick reply
[Reactor arc Iron Man keycap manual support.zip](https://github.com/5axes/CustomSupportCylinder/files/10567157/Reactor.arc.Iron.Man.keycap.manual.support.zip)

**Comment by 5axes** [comment, marked as answer]
Your supports size for the smallest of them are 0.4mm for a nozzle size of 0.4 mm. You cannot expect in this case to get the right result. Are you sure it's the right size of the model ? 18x18x15 mm

**Reply by Deadit3** [reply]
I thought I got it to work before but must be mistaking getting 1mm to work. However removing the .5mm supports and setting it to concentric seems to work still does weird things on other settings that I'd never encountered in the past. I usually have supports set to gyroid and it didn't seem to affect it in the past which is why I thought something was wrong.
![image](https://user-images.githubusercontent.com/113082660/216301661-58052218-7d6b-4640-b0d9-b51fbb9c760e.png)
Sorry for wasting your time. Thank you for helping me out here.
